decomp2gef.py

```diff
@@ -207,7 +221,33 @@ def add_native_symbols(self, sym_info_list):
 
         # locate the base address of the binary
         vmmap = get_process_maps()
-        text_base = min([x.page_start for x in vmmap if x.path == get_filepath()])
+ 
+        global downloaded
+        if is_remote_debug() and downloaded == False:
```

Comment:
see other comment about the same code. I can deal with it not being run multiple times.